tests/test_pico_project.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+
+# basic integration testing
+def test_basic_project(tmp_path):
+    test_path = str(tmp_path / "test_project")
```

Comment:
I'd do this as `test_path = os.path.join(tmp_path, "test_project")` but perhaps that's the "old-fashioned" way of doing things? :shrug:

Reply:
I'm gonna mark this as resolved, as it seems the `/` operator is also used for building paths in several other places in this repo, and it makes sense to keep the code within a repo self-consistent.